tests/Unit/Domain/CoordinateMaps/Test_Wedge2D.cpp

```diff
@@ -187,40 +187,37 @@ SPECTRE_TEST_CASE("Unit.Domain.CoordinateMaps.Wedge2D.Map", "[Domain][Unit]") {
   test_wedge2d_all_orientations(false);  // Equidistant
   test_wedge2d_all_orientations(true);   // Equiangular
   test_equality();
-  CHECK(not CoordinateMaps::Wedge2D{}.is_identity());
+  CHECK(not Wedge2D{}.is_identity());
 }
 
 // [[OutputRegex, The radius of the inner surface must be greater than zero.]]
 [[noreturn]] SPECTRE_TEST_CASE("Unit.Domain.CoordinateMaps.Wedge2D.RadiusInner",
                                "[Domain][Unit]") {
   ASSERTION_TEST();
 #ifdef SPECTRE_DEBUG
-  auto failed_wedge2d =
-      CoordinateMaps::Wedge2D(-0.2, 4.0, 0.0, 1.0, OrientationMap<2>{}, true);
+  auto failed_wedge2d = Wedge2D(-0.2, 4.0, 0.0, 1.0, OrientationMap<2>{}, true);
   static_cast<void>(failed_wedge2d);
   ERROR("Failed to trigger ASSERT in an assertion test");
 #endif
 }
 
-// [[OutputRegex, Circularity of the inner surface must be between 0 and 1]]
+// [[OutputRegex, Sphericity of the inner surface must be between 0 and 1]]
```

Comment:
Is `Circularity` being replaced with `Sphericity` globally or is this an accident? (I'm fine with keeping both, but also fine with collapsing both terms into `Sphericity` if needed, as long as we're consistent)

Reply:
This tests the error message that is printed when passing invalid values to the constructor. The error message from `Wedge<Dim>` always says `Sphericity`, whereas the previous 2D implementation said `Circularity`, so this output regex had to change.